src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Markup/Baml2006/WpfGeneratedKnownProperties.cs

```diff
@@ -6420,7 +6428,15 @@ private WpfKnownMember Create_BamlProperty_Grid_RowDefinitions()
                             false // IsAttachable
                                      );
             bamlMember.GetDelegate = delegate(object target) { return ((System.Windows.Controls.Grid)target).RowDefinitions; };
+            bamlMember.SetDelegate = delegate(object target, object value) { var cd = ((System.Windows.Controls.Grid)target).RowDefinitions;
+                                                                            var cd2 = (System.Windows.Controls.RowDefinitionCollection)value; 
```

Comment:
Similar as above

Reply:
Infact, since we have ColumnDefinitions and RowDefinitions as settable properties in this PR, it will be fine to just do 
```
((System.Windows.Controls.Grid)target).RowDefinitions = (ColumnDefinitionCollection)value;
```

Reply:
Done